preview/src/routes/index.svelte

```diff
@@ -152,11 +219,20 @@
     overflow-y: scroll;
   }
 
-  :global(#select-theme) {
-    width: 8rem;
+  :global(#select-theme),
+  :global(#select-preview-size) {
```

Comment:
How about reducing the `#select-preview-size` min-width to `4rem`?

Reply:
I was going for the same values as it is, in my opinion, more visually pleasing and consistent. This is then also reflected on mobile viewports, as both dropdowns occupy half of the container's width.

Happy to implement your suggestion if you still insist on it though.

Reply:
Ah, I didn't check mobile. Making them 1:1 on mobile is a nice touch. Is it possible to only apply the `4rem` for non-mobile viewports?

Reply:
`4rem` seems to be equal to not having the rule at all and it has no/minimal effect on common mobile viewports nonetheless. Updated to `4rem` in https://github.com/carbon-design-system/carbon-icons-svelte/pull/145/commits/6b5adaef4f24c8619592e0d24d32a847203c3c12.